modules/images-using-images-s2i-perl.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-using-images-s2i-perl_{context}"]
+= Perl overview
+
+{product-title} provides source-to-image (S2I) enabled Perl images for building and running Perl applications. The Perl S2I builder image assembles your application source with any required dependencies to create a new image containing your Perl application. This resulting image can be run either by {product-title} or by a container runtime.
+
+[id="images-using-images-s2i-perl-versions_{context}"]
+== Versions
+
+Currently, {product-title} supports versions `5.26` and `5.30` of Perl.
```

Comment:
Personally, I would get rid of this. Versions of images delivered by the Samples operator can disappear with each OpenShift release due to EOL.